src/openrct2/object/Object.cpp

```diff
@@ -301,6 +302,20 @@ std::unique_ptr<IStream> ObjectAsset::GetStream() const
     return {};
 }
 
+ObjectEntryDescriptor::ObjectEntryDescriptor(const ObjectRepositoryItem& ori)
+{
+    if (ori.Identifier != "")
```

Comment:
If `Identifier` is a string, this should be preferably:
```suggestion
    if (!ori.Identifier.empty())
```